src/app/summary/services/projects.data.ts

```diff
@@ -1,31 +1,42 @@
-import {Project} from "../../projects/project";
+import { Project } from '../../projects/project';
 
 function makeProject() {
   return {
-    addedToIndex: "",
+    addedToIndex: '',
     arrayExpressAccessions: [],
     authors: [],
-    date: "",
-    dcpUrl: "",
+    date: '',
+    dcpUrl: '',
     egaDatasetsAccessions: [],
     egaStudiesAccessions: [],
+    dbgapAccessions: [],
     enaAccessions: [],
     geoAccessions: [],
     publications: [],
-    title: "",
-    uuid: "",
+    title: '',
+    uuid: '',
     organs: [],
     technologies: [],
-    cellCount: 0
+    cellCount: 0,
   };
 }
 
 export function makeDummyProjects() {
```

Comment:
Is this for testing? What does "dummy" mean here? If it is for testing, it should be moved to a testing file.

Reply:
I did not write this, but I assume it means mock

Reply:
Let's leave it for now.